CareKitUI/CareKitUI/Shared/Task/InstructionsTaskView.swift

```diff
@@ -32,7 +32,7 @@ import SwiftUI
 
 #if os(iOS)
 private let completionButtonTextPadding: CGFloat = 14
-#elseif os(watchOS)
+#else
 private let completionButtonTextPadding: CGFloat = 8
```

Comment:
I made every OS other than iOS use this padding value, let me know if we should update it to be more specific.